ci/checks/headers_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
This looks good, but CI isn't going to run this check. You could add it to `ci/check/style.sh` if you want to it run with every PR.

Reply:
added